### PR TITLE
Books page

### PR DESCRIPTION
**Pull request by SaySayo**
Redesigned the books page according to the learn area figma redesign
Before
Screenshot 2023-10-02 at 08 47 05
After
Screenshot 2023-10-02 at 08 55 32

### CODE REVIEW

**Review by christinerose — APPROVED** [review]

**Review by christinerose — APPROVED** [review]

**Review by christinerose — APPROVED** [review]

File: src/ocamlorg_frontend/pages/books.eml [hunk 1]
```diff
@@ -2,29 +2,41 @@ let render_books books =
...
-                <div class="prose prose-orange mb-6"><%s! item.body_html %></div>
-                <% item.links |> List.iter (fun (link : Data.Book.link) -> %>
-                    <a class="btn btn-secondary" href="<%s link.uri %>">
-                        <span><%s link.description %></span>
-                        <%s! Icons.chevron_right "h-4 w-4" %>
-                    </a>
-                <% ); %>
+                <h4 class="font-bold text-2xl mb-6"><%s item.title %></h4>
+                <div class="font-normal text-base mb-6 uppercase"><%s item.authors |> String.concat ", " %></div>
+                <div class="prose-md prose-orange mb-6 md:prose-lg"><%s! item.body_html %></div>
```
Comment (sabine) [inline src/ocamlorg_frontend/pages/books.eml]:
```suggestion
                <div class="prose prose-md prose-orange mb-6 md:prose-lg"><%s! item.body_html %></div>
```

**Review by christinerose — APPROVED** [review]

File: src/ocamlorg_frontend/pages/books.eml [hunk 2]
```diff
@@ -33,34 +46,30 @@ Learn_layout.render
...
-        <div :class="language != 'portugese' ? 'hidden' : 'block'">
-            <%s! render_books (List.filter (fun x -> x.Data.Book.language = "portugese") books) %>
-        </div>
-        <div :class="language != 'japanese' ? 'hidden' : 'block'">
-            <%s! render_books (List.filter (fun x -> x.Data.Book.language = "japanese") books) %>
-        </div>
-        <div :class="language != 'chinese' ? 'hidden' : 'block'">
-            <%s! render_books (List.filter (fun x -> x.Data.Book.language = "chinese") books) %>
+<div class="bg-default dark:bg-dark-default"> 
+    <h2 class="font-bold mt-10 mb-4 md:mb-4 lg:mt-0 mb-10">Books On OCaml (15)</h2>
```
Comment (sabine) [inline src/ocamlorg_frontend/pages/books.eml]:
```suggestion
    <h2 class="font-bold mt-10 mb-6 lg:mt-0">Books On OCaml (15)</h2>  
```

I see that you took the value from Figma inspect with developer mode. But the problem was that the text element in Figma also had whitespace within itself. There is a way to automatically shrink Figma text elements: double click on their bottom border.

**Review by sabine — COMMENTED** [review]
I noticed that even when you filter, the number of books doesn't change. I expected that it lists the number of books that match the selected criteria.

Other than that looks mergeable to me.

**Comment by SaySayo** [comment]
Do you mean the number of books in the title? I.e 15? @sabine

**Comment by sabine** [comment]
Yes, the number of books in the title. It seems odd that it always says 15

**Comment by SaySayo** [comment]
Yes I agree, completely missed that. Will leaving it as "Books on OCaml" suffice? @sabine

**Comment by sabine** [comment]
Using the length of the results list doesn't seem hard. I think that would be the best to do. @SaySayo

**Review by sabine — APPROVED** [review]

**Comment by SaySayo** [comment]
Okay. Missed your later comment @sabine

**Comment by sabine** [comment]
All good, I just want to merge this so we can forget about it.

**Pull request merged**